test/IDE/print_ast_tc_decls.swift

```diff
@@ -108,7 +108,7 @@ struct d0100_FooStruct {
 // PASS_COMMON-LABEL: {{^}}struct d0100_FooStruct {{{$}}
 
   var instanceVar1: Int = 0
-// PASS_COMMON-NEXT: {{^}}  var instanceVar1: Int{{$}}
+// PASS_COMMON-NEXT: {{^}}  @_bindingHasInit var instanceVar1: Int{{$}}
```

Comment:
Can you change swift-ide-test to not print this attribute, since it will confuse users who see it in generated interfaces in Xcode?

Reply:
SourceKit generally ignores it (e.g. it isn't included in the lists of attributes in its RPC protocol thing), but generated interfaces go through a different route?

Reply:
UserInaccessible takes care of this. swift-ide-test prints all attributes by default, IIRC.